codeGenerator/configuration/configuration.xml

```diff
@@ -10,8 +10,10 @@
 </robotConfigurations>
 
 <treeviewParentNameExtensions>
-	<string>usage</string>
+	<string>name</string>
```

Comment:
I don't get removing the usage; that attribute is used throughout the robot code